docs/operator-manual/applicationset.yaml

```diff
@@ -214,6 +204,12 @@ spec:
                 - env-prod
           maxUpdate: 10%    # maxUpdate supports both integer and percentage string values (rounds down, but floored at 1 Application for >0%)
 
```

Comment:
```suggestion
# To preserve this annotation and label we can use the preservedFields property 
```

No harm adding this comment too, WDYT?

Reply:
I don't really think this comment adds anything of value, but I did improve the comments here a bit.